tracpro/static/less/trac.less

```diff
@@ -203,4 +203,44 @@ footer {
 /* Override datepicker colors */
 .ui-datepicker select.ui-datepicker-month, .ui-datepicker select.ui-datepicker-year {
   color: #000000;
-}
\ No newline at end of file
+}
```

Comment:
I've made a few improvements here. I think it's close enough to the mockup now. Let me know!